src/App.vue

```diff
@@ -12,38 +12,16 @@ GlobalSiteOptions.value = GetSiteOptions()
 </script>
 
 <template>
-	<!-- the main things i need to add:
-		TODO:
-		1) add a modal for tips on less obvious functionality
-		2) Fix clicking anywhere below the titles triggereing content editable
-		3) text font/ size change options
-		4) Have ability to set background image instead of color
-		5) Padding 
-		6) gracefully handle name collisions in creating new charts and renaming charts
-		7) improve editing of artist and album name to maybe be editable all at once?
-		8) Allow for saving as JPEG, GIF, WebM -- https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toDataURL 
-		9) look more into the vite vue plugin
-
-		A) options to set how many albums in a row
-			1) default options for top 10, top 25, top 50, top 100
-			2) have some sort of ui to create these yourself
-	-->
-
 	<!-- in case the OG site goes down a reference image -->
 	<!-- https://i.gyazo.com/9fb8f19495588fd3b31c47389d248bcb.png -->
-	<div class="flex">
-		<Sidebar class="tw-sidebar-width" @can-render-chart="renderChart = true" />
-		<Chart v-if="renderChart" class="w-4/5" />
+	<div class="flex text-white text-center font-sans antialiased">
```

Comment:
Isn't `antialiased` (that is, grayscale + regular antialias options for osx/win respectfully) a default option in most (if not all) browsers? Why the enforcement?

Just curious overall.